app/frontend/cypress/downloads/downloads.html

```diff
@@ -0,0 +1 @@
+Cr24
```

Comment:
Delete this file and maybe also add this folder to gitignore

Reply:
Removed file.